src/draw.rs

```diff
@@ -8,8 +8,10 @@ use crate::data::{FontMetrics, Workspace};
 use crate::design_space::ViewPort;
 use crate::edit_session::EditSession;
 use crate::guides::{Guide, GuideLine};
-use crate::path::{EntityId, Path, PointType};
-use druid::kurbo::{Affine, BezPath, Circle, CubicBez, Line, PathSeg, Point, Rect, Vec2};
+use crate::path::{EntityId, Path, PathSeg, PointType};
+use druid::kurbo::{
+    Affine, BezPath, Circle, CubicBez, Line, PathSeg as KurboPathSeg, Point, Rect, Vec2,
```

Comment:
Personally I prefer importing `kurbo::self` and then saying `kurbo::PathSeg`, but that's a matter of style, certainly won't block on this.